connect/src/spirc.rs

```diff
@@ -1246,7 +1245,7 @@ impl SpircTask {
             .connect_state
             .context
             .as_ref()
-            .ok_or(ConnectStateError::NoContext)?;
+            .ok_or(StateError::NoContext(false))?;
```

Comment:
I tried a enum struct here, but it did feel annoying to always specify the field if i just want to decide between two values. But because i might add an context for autoplay next. I might automatically convert this into an enum struct by which this should be more explanatory.

The value current is the shuffle value, so if the shuffle context or normal context was used.